lld/MachO/Options.td

```diff
@@ -1407,3 +1407,7 @@ def debug_variant : Flag<["-"], "debug_variant">,
     HelpText<"Do not warn about issues that are only problems for binaries shipping to customers.">,
     Flags<[HelpHidden]>,
     Group<grp_ignored_silently>;
+def no_warn_duplicate_libraries : Flag<["-"], "no_warn_duplicate_libraries">,
```

Comment:
If someone were to implement `-warn_duplicate_libraries`, then we won't want to ignore this flag anymore, right? I'd leave a comment above this line if that were the case.

Reply:
done